<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulateHomeSettings</class>
 <widget class="QWidget" name="SimulateHomeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#titleLabel {
	padding-width-left:5px;
	padding-width-right:5px;
	border: 1px solid white;
	border-radius: 10px;
	min-width: 100px;
	min-height: 20px;
}

QLabel#statusLabel {
	background-color: white;
}

 QComboBox {
	selection-background-color: transparent;
	selection-color: black;
}

 QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
 }
QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
 }

QFrame#frame_1,#frame_2  {
	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 3px;
	background-color:rgb(100, 200, 200, 50%);
}

QFrame#frame_1 QPushButton {
	min-width: 48px;
	min-height: 48px;
	max-width: 48px;
	max-height: 48px;
	border-image: url(:/Hac/round-button.png);
}

QFrame#frame_2 QPushButton {
	min-width: 28px;
	min-height: 28px;
	max-width: 28px;
	max-height: 28px;
	border-image: url(:/Hac/round-button-small.png);
}


QPushButton#upButton {
	image: url(:/Hac/up-arrow-black.png);
}
QPushButton#upButton:pressed {
	image: url(:/Hac/up-arrow-red.png);
}
QPushButton#downButton {
	image: url(:/Hac/down-arrow-black.png);
}
QPushButton#downButton:pressed {
	image: url(:/Hac/down-arrow-red.png);
}


QPushButton#startButton {
	image: url(:/Hac/right-arrow-black.png);
}
QPushButton#startButton:pressed {
	image: url(:/Hac/right-arrow-red.png);
}
QPushButton#stopButton {
	image: url(:/Hac/stop.png);
}
QPushButton#stopButton:pressed {
	image: url(:/Hac/stop.png);
}

QSpinBox {
	margin: 1px;
	selection-background-color: transparent;
	text: black;
	selection-color: black;
	font: normal 15px;
	border: 1px solid black;
	padding-width-right: 40px;
}

QSpinBox::up-button {
	subcontrol-origin: border;
    subcontrol-position: right center;
	width: 20px;
	height: 20px;
}

QSpinBox:up-arrow {
     image: url(:/Hac/up-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::up-arrow:pressed {
     image: url(:/Hac/up-arrow-red.png);
}


QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: right center;
	left: -20px;
	width: 20px;
	height: 20px;
}

QSpinBox::down-arrow {
     image: url(:/Hac/down-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::down-arrow:pressed {
     image: url(:/Hac/down-arrow-red.png);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Stage &gt; Mode Settings &gt; Simulate Home Setup</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>19</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Light On</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="HacLineEdit" name="lightOnHourLineEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Hour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="HacLineEdit" name="lightOnMinuteLineEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Minute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Light Off</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <item>
          <widget class="HacLineEdit" name="lightOffHourLineEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Hour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="HacLineEdit" name="lightOffMinuteLineEdit">
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Minute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>36</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="downButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="upButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>36</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_1">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="startButton"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="stopButton"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HacLineEdit</class>
   <extends>QLineEdit</extends>
   <header>haclineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/hac.qrc"/>
 </resources>
 <connections/>
</ui>
